src/optimizer/query_graph.c

```diff
@@ -8699,6 +8704,14 @@ qo_discover_sort_limit_nodes (QO_ENV * env)
   return;
 
 abandon_stop_limit:
+  /* remove error which is occured in this function */
+  if (pt_has_error (QO_ENV_PARSER (env)))
+    {
+      pt_reset_error (QO_ENV_PARSER (env));
```

Comment:
In line 8617, assert_release() is executed and the error setting is removed even though it is coming?
Or is it ignoring the case because it is unlikely to happen?

Reply:
This routine checks if sort limit optimization is possible. If an error occurs in this function, only sort limit optimization should be disabled, and it should not have any other effect.